x-pack/plugins/security_solution/public/cases/components/case_view/helpers.ts

```diff
@@ -1,41 +0,0 @@
-/*
```

Comment:
git being weird. Said you deleted it and remade it 🤷🏾

Reply:
you're right i think im going to rename it back. i think i deleted whatever needed it to be tsx